### DISCUSSION TITLE
I hope the file upload function supports Cloudflare R2 storage buckets.

### DISCUSSION THREAD

**Discussion by FoehnWorknet** [danny-avila/LibreChat#3833] [Feature Requests & Suggestions]
### What features would you like to see added?

![Cloudflare-R2-Storage-OG](https://github.com/user-attachments/assets/5c3c1b93-83af-421d-b0fb-44e725539ef3)

### More details

https://dash.cloudflare.com/

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by robsoncombr** [comment]
I would believe this shoud be part of one-single implementation "S3 Compatible Objects Storage Support"
We have another opened that looks related to me https://github.com/danny-avila/LibreChat/issues/2898
